### ISSUE TITLE
Play button is inaccessible to keyboard focus

### ISSUE THREAD

**Issue by groupboard** [groupboard/ytdefer#2]

**Comment by djmtype** [comment]
Since a `<button aria-label="Play [title of video]"></button>` should to be added in place of the button png images, it might be preferred to use base64 svgs as a background image. So you can basically use the same image twice, just change the `fill` color. It may be only a small victory, but 600 bytes total versus 8,600 bytes total for both play button images is still a win.

Black button: 

```
background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='60' height='42.46' viewBox='0 0 60 42.46'%3E%3Ctitle%3Eyoutube%3C/title%3E%3Cpath d='M39.13,20.9,22.72,12.59a.66.66,0,0,0-1,.58V29.31a.66.66,0,0,0,.94.6l16.41-7.83A.66.66,0,0,0,39.13,20.9Z' transform='translate(0 0)' style='fill:%23fff'/%3E%3Cpath d='M47.54,0H12.46A12.46,12.46,0,0,0,0,12.46V30A12.46,12.46,0,0,0,12.46,42.46H47.54A12.46,12.46,0,0,0,60,30V12.46A12.46,12.46,0,0,0,47.54,0ZM39.11,22.08,22.7,29.91a.66.66,0,0,1-.94-.6V13.17a.66.66,0,0,1,1-.58L39.13,20.9A.66.66,0,0,1,39.11,22.08Z' transform='translate(0 0)' style='fill:%23000000'/%3E%3C/svg%3E");
```

Red button: 

Same as above except change the `fill` style to `fill:%23f61c0d`.

**Comment by groupboard** [comment]
Fixed
